index.js

```diff
@@ -20,8 +20,8 @@ var wrap = require('ansi-wrap');
  * console.log(colors.bgblack('some string'));
  * ```
  *
- * @param  {String} `str` String to wrap with ansi codes.
- * @return {String} Wrapped string
+ * @param  {string} message String to wrap with ansi codes.
```

Comment:
> `str` vs. message simply confused the intellisense implementation in VS Code

Is this because of the backticks or the word `str`? If it's not due to the backticks, will you add those around `message`?

Reply:
Combination of both.
Backticks break it likely because it doesn't stripe them out before matching them against function parameter. Standard supports the usage of `-` to break it apart from the description, if thats the goal here.
`str` simply doesn't match the parameter name, and so is ignored. Though some systems might get around this, but it would be more down to them falling back on an order based behaviour.

Reply:
@Silic0nS0ldier thanks for the information! The parameter name mismatch makes sense.

I usually use [verb](https://github.com/verbose/verb/tree/dev) along with a helper to generate the API docs in my readmes which expects the backticks around the name. Since I'm adding the API docs manually here, I'll use this syntax for the typescript support.

Reply:
Huh. Any chance you could point me in the direction of the project responsible for generating the API docs? It'd be an interesting challenge to see if I can remove the need for the helper.

And to clarify, the TypeScript definition support itself doesn't need that particular change, its just that most jsdoc compliant editors can't properly provide parameter details when using this non-standard form. EG: (cursor is over the `message` parameter, when using function later, that parameter will have the same doc popup) 
![image](https://user-images.githubusercontent.com/17376090/36357510-23150ade-1553-11e8-93ab-ddefbd78f3ec.png)